mitodl/open-discussions
# Added api to report posts and comments (#197)

#### What are the relevant tickets?
Fixes #197

#### What's this PR do?
Adds an api for reporting posts and comments to moderators

#### How should this be manually tested?
Go to /api/v0/reports/ and POST a new report (e.g. `{'comment_id': 'ID', 'reason': 'spam'} and verify in the reddit UI that this shows up for your moderator user under the reports.

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/3orieQMr5vIKM04VvW/giphy.gif)
